pages.ja/common/git.md

```diff
@@ -0,0 +1,30 @@
+# git
+
+> 分散型バージョン管理システム
+> いくつかのサブコマンドがあります。例えば `commit`, `add`, `branch`, `checkout`, `push`, などです。
+> これらには使用方法についての独自のドキュメントがあり、 `tldr git subcommand` で見ることができます。
```

Comment:
```suggestion
> いくつかのサブコマンドがあります。例えば `commit`, `add`, `branch`, `checkout`, `push`, などです。 これらには使用方法についての独自のドキュメントがあり、 `tldr git subcommand` で見ることができます。
```
We generally only support only 3 lines of page descriptions with very few exceptions. For translations, we generally try to be uniform with the main page. So I think concatenating the lines would be better.